lib/android-helpers.js

```diff
@@ -273,7 +273,9 @@ helpers.reinstallRemoteApk = async function (adb, localApkPath, pkg,
 
 helpers.installApkRemotely = async function (adb, opts) {
   let {app, appPackage, fastReset, androidInstallTimeout} = opts;
-
+  if (androidInstallTimeout ==null) {
```

Comment:
it's better to say `_.isNull(androidInstallTimeout)`, but that may not do exactly what we want. Probably more like if it's null or undefined.

Reply:
Can you elaborate on this? Does .isNull and == null have different behavior? To be honestly I've never written a line of Java in my life so I'm not very familiar to all the syntax and language specific nuances. So I'm always open to suggestions